part1/random.js

```diff
@@ -0,0 +1,18 @@
+const btn = document.querySelector('button');
+
+const widthValue = document.querySelector(".width");
+
+const heightValue = document.querySelector(".height");
+
+widthValue.setAttribute("value" , "")
+heightValue.setAttribute("value" , "");
+
+const imgDiv = document.querySelector("div.image");
+const image = document.createElement("img");
+imgDiv.appendChild(image);
+
+const createImg = function(){
+    image.setAttribute("src" , "https://source.unsplash.com/random/"+widthValue+"*"+heightValue)

```

Comment:
It should look like this:
```suggestion
  image.setAttribute("src" , 'https://source.unsplash.com/random/'+widthValue.value+"x"+heightValue.value)
```